homework3/semi-uber/UI/src/app/regestration-page/regestration-page.component.ts

```diff
@@ -40,4 +45,33 @@ export class RegestrationPageComponent implements OnInit {
   get username() {
     return this.userData.get("username");
   }
+
+  getRegForm() {
+    const inputs = this.userData.value;
+    const name = inputs.name;
+    const username = inputs.username;
+    const password = inputs.password;
+    const email = inputs.email;
+    const status = inputs.status;
+
+    const userData = [name, username, password, email, status];
+    userData.map(e => {
+      if (e === undefined) {
```

Comment:
You do not need this check, you can just add `required` Validator to all of the fields and that will be enough.